monitoring/opencensus/noxfile.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2021 Google LLC
```

Comment:
Please delete this file. CI will copy it into the directory from the base of the repository.

Reply:
Done, thank you.